google-cloud-spanner/src/main/java/com/google/cloud/spanner/MultiplexedSessionDatabaseClient.java

```diff
@@ -316,7 +327,18 @@ && verifyErrorMessage(
     }
   }
 
-  private boolean verifyErrorMessage(SpannerException spannerException, String message) {
+  boolean maybeMarkUnimplementedForPartitionedOps(SpannerException spannerException) {
+    if (spannerException.getErrorCode() == ErrorCode.UNIMPLEMENTED
```

Comment:
The backend does not throw UNIMPLMENETED now right? I think it now returns INVALID_ARGUMENT.
If that is correct, then we should remove error code check now and add it back once backend throws UNIMPLEMENTED.
With this change we are not really falling back since this check will always fail.
Correct me if i am missing something